### ISSUE TITLE
Kibana Timelion error

### ISSUE THREAD

**Issue by dcbeckett** [StamusNetworks/SELKS#273]
Getting a timelion error on kibana, with the latest nginx config https://github.com/StamusNetworks/SELKS/wiki/Kibana-did-not-load-properly  (I added in the extra config needed for /35949/)

I'm not experienced enough with kibana to know how to solve this i.e. if its a selk nginx proxy issue or not, but it appears to be related to /api/timelion/run

![Screenshot from 2020-11-16 18-24-20](https://user-images.githubusercontent.com/6516519/99292616-618fe800-2839-11eb-9ac1-4cffb9c56c30.png)

ii  elasticsearch                         7.10.0                              amd64        Distributed RESTful search engine built for the cloud
ii  elasticsearch-curator                 5.8.1                               amd64        Have indices in Elasticsearch? This is the tool for you!\n\nLike a museum curator manages the exhibits and collections on display, \nElasticsearch Curator helps you curate, or manage your indices.
ii  evebox                                1:0.12.0                            amd64        no description given
ii  kibana                                7.10.0                              amd64        Explore and visualize your Elasticsearch data
ii  kibana-dashboards-stamus              2020042401                          amd64        Kibana 6 dashboard templates.
ii  logstash                              1:7.10.0-1                          amd64        An extensible logging pipeline
ii  moloch                                2.4.2-1                             amd64        Moloch Full Packet System
ii  scirius                               3.5.0-3                             amd64        Django application to manage Suricata ruleset
ii  suricata                              1:2020100901-0stamus0               amd64        Suricata open source multi-thread IDS/IPS/NSM system.

**Comment by pevma** [comment]
Thanks for trying it out.
I think this is due to a stricter checks in 7.10+ in terms of queries.

ex:
this query errors 
```
.es(q='event_type:flow AND proto:TCP').label('TCP Flows"), .es(q='event_type:flow AND proto:UDP').label('UDP Flows')
```

but this does not
```
.es(q="event_type:flow AND proto:TCP").label("TCP Flows"), .es(q="event_type:flow AND proto:UDP").label("UDP Flows")
```

I will be pushing  anew dashboards pkg soon - by end of week.
Would you be wiling to give it a try and feedback?

**Comment by dcbeckett** [comment]
Ah I did see this ticket earlier - https://github.com/elastic/kibana/issues/83297 and I was wondering if it was related, I also wonder if they will fix it which may negate your need to fix it, but either way I'm happy to test if required

**Comment by ManuelFFF** [comment]
Hi all,

I was facing the same issue after upgraded to ELK v7.10. This is not a SELKS issue, but an ELK issue. The new parser for Timelion got stricter. ELK is not planning to fix anything since this is not considered an issue, but an improovement. User need to update their Timelion queries. SELKS could provide an update having all queries fixed, but user must fix custom queries by themselves. Please check my own post below, hopping it can help.

**Fix:** Replace all single quotes `'` by double quotes `"` in the Timelion queries.

https://discuss.elastic.co/t/timelion-7-10-error-syntax-error/256016

Cheers!

**Comment by dcbeckett** [comment]
Seems like they are fixing it now in Elastic search

https://github.com/elastic/kibana/pull/84196

**Comment by ManuelFFF** [comment]
Yeah. It looks like they will. I'm glad for all users and myself!

Cheers!

**Comment by pevma** [comment]
Good to hear, thanks for the feedback and update.

**Comment by pevma** [comment]
Seems all fixed in Kibana 7.10.1
